cli/cmd/stat.go

```diff
@@ -224,19 +224,21 @@ func writeStatsToBuffer(resp *pb.StatSummaryResponse, reqResourceType string, w
 	}
 
 	lastDisplayedStat := true // don't print a newline after the final stat
```

Comment:
can you move `lastDisplayedStat` under `case k8s.All`?
also tioli: i struggled to parse the meaning of this variable, does `firstDisplayedStat` make sense?
```
firstDisplayedStat := true // don't print a newline before the first stat
```